controllers/component_build_controller_pac.go

```diff
@@ -1089,7 +1089,7 @@ func generateCelExpressionForPipeline(component *appstudiov1alpha1.Component, gi
 	// Set path changed event filtering only for Components that are stored within a directory of the git repository.
 	// Also, we have to rebuild everything on push events, so applying the filter only to pull request pipeline.
 	pathChangedSuffix := ""
-	if onPull && component.Spec.Source.GitSource.Context != "" && component.Spec.Source.GitSource.Context != "/" {
+	if onPull && component.Spec.Source.GitSource.Context != "" && component.Spec.Source.GitSource.Context != "/" && component.Spec.Source.GitSource.Context != "./" {
```

Comment:
if context dir would be specified just with '.' I'd say it would cause some other issues

but './' is default what is UI adding

Reply:
> if context dir would be specified just with '.' I'd say it would cause some other issues

What sort of issues? Using `.` would be my go-to way to do this

Reply:
I am not sure that is just my guess.

I can add '.' there as well, what  others think?

Reply:
Iif `.`  works as expected, I would incorporate it here.

Reply:
@rcerven could you check if `.` works? If there is an issue with it, let's exclude as well.

Reply:
ok I'll add it